### ISSUE TITLE
swe_utc_to_jd returns only ET (TT) & not an array containing both TT & UT

### ISSUE THREAD

**Issue by iruleee** [vm75/sweph.dart#11]
Thanks a lot for this flutter package.
The function `Sweph.swe_utc_to_jd` returns only ET (TT) & not an array containing both TT & UT. 
According to Swiss Ephemeris documentation, it should return
```
dret               /* return array, two doubles:
                   * dret[0] = Julian day in ET (TT)
                   * dret[1] = Julian day in UT (UT1) */
```
Thanks a lot

**Comment by vm75** [comment]
Thanks for reporting this @iruleee. I will debug this.

**Comment by vm75** [comment]
Fixed in version 2.10.3+16
